utils/src/main/java/com/cloud/utils/ssh/SshHelper.java

```diff
@@ -206,4 +216,87 @@ public static void scpTo(String host, int port, String user, File pemKeyFile, St
                 conn.close();
         }
     }
+
+    /**
+     * It gets a {@link Session} from the given {@link Connection}; then, it waits
+     * {@value #WAITING_OPEN_SSH_SESSION} milliseconds before returning the session, given a time to
+     * ensure that the connection is open before proceeding the execution.
+     *
+     * @param conn
```

Comment:
@GabrielBrascher this is the kind of javadoc I don't like. The header adds an explanation but these tags don't add value to the names of the params themselves. (just saying, no biggy)